core/models/quota.py

```diff
@@ -107,18 +136,25 @@ def default_dict(cls):
             'cpu': cls._meta.get_field('cpu').default(),
             'memory': cls._meta.get_field('memory').default(),
             'storage': cls._meta.get_field('storage').default(),
+            'floating_ip_count': cls._meta.get_field('floating_ip_count').default(),
+            'port_count': cls._meta.get_field('port_count').default(),
+            'instance_count': cls._meta.get_field('instance_count').default(),
             'storage_count': cls._meta.get_field('storage_count').default(),
-            'suspended_count': cls._meta.get_field('suspended_count').default()
+            'snapshot_count': cls._meta.get_field('snapshot_count').default(),
         }
 
     class Meta:
         db_table = 'quota'
         app_label = 'core'
-        unique_together = ("cpu", "memory", "storage", "storage_count",
-                           "suspended_count")
+        # unique_together = ("cpu", "memory", "storage",
+        #                    "floating_ip_count",
+        #                    "port_count",
+        #                    "instance_count",
+        #                    "snapshot_count",
+        #                    "storage_count")
```

Comment:
FYI - Removed the unique_together command because production DB failed with "duplicate keys". We can add this in a future migration if desired, after massaging the production database.